ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -92,6 +96,14 @@ public final class SystemColumns {
           true,
           true,
           true
+      ),
+      PseudoColumn.of(
+          ROWID_NAME,
+          ROWID_TYPE,
+          ROWID_PSEUDOCOLUMN_VERSION,
+          true,
+          true,
```

Comment:
Also double-checking whether this is intentional: this currently says that users are not allowed to specify ROWID as part of INSERT VALUES statements. Is that intended? 

Maybe my questions so far boil down to: I'm confused about the intended usage of ROWID. Is it meant to be read-only, and users still have to specify the key by explicitly declaring a key column for their stream/table and populating it that way, or is ROWID meant to enable "keyless" streams/tables (from the perspective of the ksqlDB schema) which still have bytes populated in their Kafka key records?